docs/choreolib/getting-started.md

```diff
@@ -0,0 +1,165 @@
+
+    === "Java"
+
+        ```java title="Drive.java"
+        public class Drive extends SubsystemBase {
+            private final PIDController xController = new PIDController(5.0, 0.0, 0.0);
+            private final PIDController yController = new PIDController(5.0, 0.0, 0.0);
+            private final PIDController headingController = new PIDController(5.0, 0.0, 0.0);
+
+            public Drive() {
+                // Other subsystem initialization code
+                // ...
+                headingController.enableContinuousInput(-Math.PI, Math.PI);
+            }
+
+            public void followTrajectory(SwerveSample sample) {
+                // Get the current pose of the robot
+                Pose2d pose = getPose();
+
+                // Generate the next speeds for the robot
+                ChassisSpeeds speeds = new ChassisSpeeds(
+                    sample.vx + xController.calculate(pose.getX(), sample.x),
+                    sample.vy + xController.calculate(pose.getX(), sample.y),
+                    sample.omega + xController.calculate(pose.getRotation().getRadians(), sample.heading)
+                );
+
+                // Apply the generated speeds
+                driveFieldRelative(speeds);
+            }
+        }
+        ```
+
+    === "C++"
+
+        ```cpp
+        // TODO
+        ```
+
+    === "Python"
+
+        ```python
+        # TODO
+        ```
+
+=== "Differential (Tank)"
+
+    The `DifferentialSample` ([Java](/api/choreolib/java/choreo/trajectory/DifferentialSample.html), [C++](/api/choreolib/cpp/classchoreo_1_1DifferentialSample.html)) class represents a single differential drive sample along a trajectory. This example applies a given sample's velocity to the drivetrain, as well as additional feedback based on the robot's distance from the sample to "push" the robot back to the trajectory, if it is off-course.
+
+    === "Java"
+
+        ```java title="Drive.java"
+        public class Drive extends SubsystemBase {
+            private final LTVUnicycleController controller = new LTVUnicycleController(0.02);
+            private final DifferentialDriveKinematics kinematics = new DifferentialDriveKinematics(0.7);
+
+            public void followTrajectory(DifferentialSample sample) {
+                // Get the current pose of the robot
+                Pose2d pose = getPose();
+
+                // Calculate the velocity feedforward specified by the sample
+                ChassisSpeeds ff = kinematics.toChassisSpeeds(
+                    new DifferentialDriveWheelSpeeds(sample.vl, sample.vr)
+                );
+
+                // Generate the next speeds for the robot
+                ChassisSpeeds speeds = controller.calculate(
+                    pose,
+                    sample.getPose(),
+                    ff.vxMetersPerSecond,
+                    ff.omegaRadiansPerSecond
+                );
+
+                // Apply the generated speeds
+                drive(speeds);
```

Comment:
I feel this is less common for diffy drives, maybe make the example us Left and Right speeds?